tests/test_initialization.py

```diff
@@ -1194,6 +1194,73 @@ def test_lora_use_dora_with_megatron_core_raises(self):
         with pytest.raises(ValueError, match="DoRA does not support megatron_core"):
             LoraConfig(target_modules=["linear"], use_dora=True, megatron_config=megatron_config)
 
+    def test_bdlora_lora_a_block_diagonal(self, data):
```

Comment:
I think this test and the next one can be scrapped, this should be covered pretty much by the `test_custom_models.py` tests. Also, they are not really about initialization.